lib/cocoapods-downloader/base.rb

```diff
@@ -121,6 +121,24 @@ def checkout_options
         raise 'Abstract method'
       end
 
+      # Returns a User-Agent string that itentifies http network requests as
+      # originating from CocoaPods.
+      #
+      # @param  [String] version
+      #         the current cocoapods version, defaulted to Pod::VERSION.
+      #
+      # @return [String] the User-Agent string
+      #
+      def self.user_agent_string version = Pod::VERSION
+        "cocoapods/#{version}"
+      end
+
+      # @return [String] Shell command to add the custom User-Agent.
```

Comment:
why does this say it's a shell command? I thought `-A` accepts a string to specify as the user agent

this also seems specific to `Http` because it's only used for `cURL`, so maybe it shouldn't be in `Base` at all